test/unit_tests/graphql/test_processes.py

```diff
@@ -37,7 +37,7 @@ def get_processes_query(
     after: int = 0,
     filter_by: Union[list[str], None] = None,
     sort_by: Union[list[dict[str, str]], None] = None,
-) -> str:
+) -> bytes:
```

Comment:
Wondering why you changed to this?

Reply:
I noticed there were a bunch of deprecation warnings running the tests from httpx/starlette/test client(?) like `DeprecationWarning: Use 'content=<...>' to upload raw bytes/text content.` Because of the use of the `data` keyword argument. Figured I would change some of these to use the suggested `content` keyword -- though the `test_client` fixture is used everywhere in the tests and it may be better to just suppress the warning

Reply:
ohh, didn't notice the deprecation warnings, do find it better to fix it then suppress, so this is fine 👍